### DISCUSSION TITLE
[Solved]Multiple Payloads / Packages of Crazyflie 2.1

### DISCUSSION THREAD

**Discussion by seunghyunnn** [bitcraze/discussions#111] [Q&A] [answered]
Hello! 
I want my crazyflie2.1 to carry multiple packages(really lightweight stuff like clips) and drop each package at different spots. 
Has anyone had any idea on how to implement this function, considering that the maximum payload weight that a crazyflie can handle is about 15g? 
I believe that I need a really small and light weighted micro controller, but any other suggestions with/without microcontroller will be really appreciated. 
Thank you so much.

**Comment by krichardsson** [comment, marked as answer]
Hi!

We implemented a simple drop mechanism when making the [2016 Christmas video](https://www.youtube.com/watch?v=KrPWIzWcO8w). The idea was to hang the object in a very thin wire and when we wanted to release it, we passed a high current through the wire to burn it off. Obviously this requires quite some work between drops to install a new wire. 

Maybe not the most convenient method, but it was fun to implement! We based the design on a modified LED-ring deck where we removed the front LEDs and connected the thin wire instead, we used a strand of steel wool. All we had to do to release the object was to set the [ring.headlightEnable](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/params/#ringheadlightenable) parameter to 1.
 
At around 39 seconds into the video you can even see a small flash (going down left from the Crazyflie) when a melted part of the wire is falling off.

**Reply by seunghyunnn** [reply]
Thank you so much for your reply!
I've never thought of burning the steel wool but I guess it is a really good idea! I just checked the video that you attached and it is so cool and nice. Thank you so much. 

Can I ask two follow-up questions? 
1. The drone in the video seems stable at the moment it drops the ornament, and after it drops the ornament. Are there special things you have done to maintain the crazyflie's stability? 
2. Moreover, how did you position the drone precisely? In the video, it seems like your crazyflie is dropping the garments to the tree from the precise position that you have intended to drop them. As I had some issues with precise positioning, it will be greatly appreciated if you could share what extra effort you put into driving the crazyflie with such a good precision. 

Thank you so much again for your suggestion and detailed instruction:)

**Reply by krichardsson** [reply]
The good precision is mainly handled by creative cutting of the video. Never trust what you see on the internet ;-)

We used the [Loco positioning system](https://www.bitcraze.io/documentation/system/positioning/loco-positioning-system/) for the video and it is generally wobbling around a bit, at least too much to manage to drop the ball on a branch (unfortunately). The dropping was actually done by hand, outside the picture.
It is possible that the [lighthouse system](https://www.bitcraze.io/documentation/system/positioning/ligthouse-positioning-system/) (or a [mocap system](https://www.bitcraze.io/documentation/system/positioning/mocap-positioning/)) is good enough to do it, but it wouldn't be easy.

**Reply by seunghyunnn** [reply]
Oh, I see. That solves all of my questions! Thank you so much for your reply:) 
I was currently using the lighthouse system. 
Thank you so much.

**Comment by knmcguire** [comment]
@seunghyunnn here in discussions you can mark an answer to be the correct one :) that is what we do now instead of the [solved] prefix (we used to do that in the old forum).

I've now selected @krichardsson answer for you now if that is okay

**Comment by derekdon** [comment]
I was thinking would a mini electromagnet work? As in you just switch it off over the target to drop a payload. Not sure if you could repurpose one of the existing decks, mini electromagnet and a downward facing camera. 

payload - string - metal washer end | magnet on deck - crazyfile. 

I would like to get a payload system working too.

Saw this: https://www.youtube.com/watch?v=Kcz1ZxG_IlQ

**Reply by ArisMorgens** [reply]
Hello @derekdon!

This sounds like a really fun project!
A mini electromagnet should work with the proper deck design. If you want some starting points, feel free to open a new discussion. Good luck with your project!

**Reply by derekdon** [reply]
Finding an electromagnet that's light enough is an issue. 

Was thinking a latch might work.
https://www.aliexpress.com/item/1005004961248588.html

But that would only enable dropping... no pickup.

I'll keep looking for parts.

**Reply by derekdon** [reply]
I'll start a separate discussion to get some advice on how to achieve this. I'm new to the crazyflie platform.

**Reply by ArisMorgens** [reply]
Great! We are more than happy to help you understand the platform and make progress!
